docker/docker-compose.yml

```diff
@@ -1,18 +1,15 @@
 # Note that mongo container is started before Orion, as Orion needs it
 # as dependency and reserve the order has been found problematic in some
 # low resource hosts
-
-# Uncomment the --smallfiles line if you host doesn't have too much free space
 mongo:
```

Comment:
This should be aligned with https://github.com/telefonicaid/fiware-orion/blob/develop/docker/README.md#run-a-container-pulling-an-image-from-the-cloud-recommended. In particular, apply the same "uncomment" to smallfiles and add the `--nojournal`.